kroxylicious-kms-provider-aws-kms-test-support/src/main/java/io/kroxylicious/kms/provider/aws/kms/AbstractAwsKmsTestKmsFacade.java

```diff
@@ -147,20 +147,17 @@ public void deleteKek(String alias) {
         @Override
         public void rotateKek(String alias) {
             var key = read(alias);
+
```

Comment:
Can we move the explanation about why we need the alternative approach for localstack to here?

Reply:
Yes